src/entity/active_enum.rs

```diff
@@ -275,6 +275,64 @@ mod tests {
         assert_eq!(format!("{}", DeriveCategory::Small), "'S'");
     }
 
+    #[test]
+    fn display_mode() {
+        // Completely default -> All print values
+        #[derive(PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+        #[sea_orm(rs_type = "i32", db_type = "Integer")]
+        enum DefaultDisplay {
+            One = 1,
+            Two = 2,
+            Three = 3,
+        }
+
+        assert_eq!(format!("{}", DefaultDisplay::One), "1");
+        assert_eq!(format!("{}", DefaultDisplay::Two), "2");
+        assert_eq!(format!("{}", DefaultDisplay::Three), "3");
```

Comment:
As per our discussion, I think we should remove the default Display impl, resulting in a breaking change.

Reply:
done in 656c777e8c7d33d2eb4f316be3b7627f07d5c72f

Reply:
done in 93cdf20277d1cc4294bf5a8301b5daff6a2639f4